extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureUtils.java

```diff
@@ -40,11 +40,26 @@
 public class AzureUtils
 {
 
+  public static final String DEFAULT_AZURE_ENDPOINT_SUFFIX = "core.windows.net";
   @VisibleForTesting
   static final String AZURE_STORAGE_HOST_ADDRESS = "blob.core.windows.net";
+  public static final String DEFAULT_AZURE_BLOB_STORAGE_ENDPOINT_SUFFIX = "blob." + DEFAULT_AZURE_ENDPOINT_SUFFIX;
+  private final String blobStorageEndpointSuffix;
+
+  /**
+   * Creates an AzureUtils object with the blob storage endpoint suffix.
+   *
+   * @param blobStorageEndpointSuffix the blob storage endpoint, like <code>"blob.core.windows.net"</code>,
+   *                                  <code>"blob.core.chinacloudapi.cn"</code> or
+   *                                  <code>"blob.core.usgovcloudapi.net</code>"
+   */
+  public AzureUtils(String blobStorageEndpointSuffix)
```

Comment:
since its a utils class, lets not add a constructor here.

Reply:
done